packages/cli/create-strapi-app/templates/example/package.json

```
@@ -4,7 +4,8 @@
     "start": "strapi start",
     "build": "strapi build",
     "strapi": "strapi",
-    "deploy": "strapi deploy"
+    "deploy": "strapi deploy",
+    "seed-example": "node ./scripts/seed.js"
```

```suggestion
    "seed:example": "node ./scripts/seed.js"
```
to follow our normal pattern, could add other seed data in the future

---

why not